src/main/java/seedu/address/logic/commands/addcommand/AddIssueCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands.addcommand;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+public class AddIssueCommand extends AddCommand {
+
+    /* Possible Fields: */
+    private Issue issue;
+
+    public AddIssueCommand(Issue issue) {
+        requireNonNull(issue);
+        this.issue = issue;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        try {
+            IssueList issueList = model.getIssueList();
```

Comment:
Just a note for me too: For now Issue methods are not exposed on Model API, because they are self contained. We could change this later, but Issue is not a priority atm.